<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoggedInDialog</class>
 <widget class="QDialog" name="LoggedInDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}
QCommandLinkButton
{
}
QDialog
{
background: QLinearGradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #eef, stop: 1 #ccf);
border-width: 1px;
border-color: #339;
border-style: solid;
border-radius: 7;
padding: 3px;
font-size: 10px;
padding-left: 5px;
padding-right: 5px;

}

QPushButton {
color: white;
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #28d, stop: 0.1 #39e, stop: 0.49 #47c, stop: 0.5 #66b, stop: 1 #77c);
border-width: 1px;
border-color: #339;
border-style: solid;
border-radius: 7;
padding: 3px;
font-size: 10px;
padding-left: 5px;
padding-right: 5px;
min-width: 100px;
max-width:100px;
min-height: 20px;
max-height: 20px;
}</string>
  </property>
  <widget class="QPushButton" name="builderAndPusherPushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>260</y>
     <width>112</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Image Manager</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cloudProviderAccountSetupPushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>112</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cloud Provider</string>
   </property>
  </widget>
  <widget class="QLabel" name="iconLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>231</width>
     <height>211</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/images/aeolus-logo.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logOutPushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>300</y>
     <width>112</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>LogOut</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
